api_proxy.yaml

```diff
@@ -442,6 +503,21 @@ definitions:
     required:
       - items
       - page_size
+  Services:
+    type: object
+    title: A list of services
+    properties:
+      items:
+        type: array
+        items:
+          $ref: "#/definitions/ServiceId"
```

Comment:
would it be possible to return the tuple `(serviceId, version)` ? this would make it easier for the app to cache the results

Reply:
yes

Reply:
ok pls add a story, we'll do it later, we'll use just the `serviceId` for now

Reply:
https://github.com/teamdigitale/digital-citizenship-functions/pull/330